<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>120</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>dcm2</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item row="0" column="0">
     <widget class="QCheckBox" name="compressCheck">
      <property name="maximumSize">
       <size>
        <width>89</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="toolTip">
       <string>If checked, images will be stored as compressed .nii.gz</string>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>Compress</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>87</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>87</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string> Output name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="outputFilenameEdit">
      <property name="minimumSize">
       <size>
        <width>231</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>231</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Name of NIfTI images (%c= comments, %f folder name, %n name, %p protocol, %s series, %t time)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>96</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>96</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string> Output folder</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="folderButton">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Set location for converted images (or choose to have files saved to the same location as the input images)</string>
      </property>
      <property name="text">
       <string>Output Folder</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="5">
     <widget class="QTextEdit" name="theTextView">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="autoFormatting">
       <set>QTextEdit::AutoBulletList</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>760</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDcm2">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionDICOM_to_NIfTI"/>
    <addaction name="actionPAR_REC_to_NIfTI"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
   </widget>
   <addaction name="menuDcm2"/>
   <addaction name="menuEdit"/>
  </widget>
  <action name="actionDICOM_to_NIfTI">
   <property name="text">
    <string>DICOM to NIfTI...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionPAR_REC_to_NIfTI">
   <property name="text">
    <string>PAR/REC to NIfTI...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
